PROJECT

```diff
@@ -99,4 +99,18 @@ resources:
     defaulting: true
     validation: true
     webhookVersion: v1
+- api:
+    crdVersion: v1
+    namespaced: true
+  controller: true
+  domain: emqx.io
+  group: apps
+  kind: EmqxRebalance
+  path: github.com/emqx/emqx-operator/apis/apps/v1beta4
+  version: v1beta4
+  webhooks:
+    conversion: true
+    defaulting: true
```

Comment:
I think we doesn't need conversion webhook and defaulting webhook

Reply:
> I think we doesn't need conversion webhook and defaulting webhook

agree with you